app/controllers/adopted_controller.rb

```diff
@@ -0,0 +1,35 @@
+class AdoptedController < ApplicationController
+  before_action :authenticate
+
+  def index
+    @things = Thing.where.not(user_id: !nil)
```

Comment:
I don't think you want `!nil`, but just `nil`

Interestingly --  not that anyone cares -- `!nil` evaluates to `true` and `true` gets converted to `1` in the sql, so you end up with `WHERE user_id != 1`. (I only know this because I checked the console)